analyzers/golang/analyze.go

```diff
@@ -23,6 +23,11 @@ func (a *Analyzer) Analyze() (graph.Deps, error) {
 	m := a.Module
 	log.Debugf("%#v", m)
 
+	switch a.Options.Strategy {
+	case "lockfile:gomodules":
```

Comment:
why "gomodules" and not "gomod" or "go.mod"? Or do you intend to combine the deps listed in both files as the "gomodules" strategy?

Reply:
Good question. For one, I could probably have left this part out, I'm not entirely sure what this will look like until we do fallbacks. The only reason I did this is to follow the pattern of the other strategies which are `manifest:gomodules`, `manifest:dep`, etc. I think `go.mod` fits more with the concept of a manifest file and `go.sum` with a lockfile so I will also be changing that.

Reply:
Your comment brings up a good point though, I'm going to remove this block.